engines/native/sandbox.go

```diff
@@ -76,6 +86,39 @@ func newSandbox(b *sandboxBuilder) (*sandbox, error) {
 	return s, nil
 }
 
+func fetchContext(context, destdir string) error {
+	filename, err := util.Download(context, destdir)
+	if err != nil {
+		return fmt.Errorf("Error downloading '%s': %v", context, err)
+	}
+
+	// TODO: verify if this will harm Windows
+	if err = os.Chmod(filename, 0700); err != nil {
```

Comment:
Add a TODO to abstract this away with a method in the system package... We'll have to add such a method, I'm okay with doing that later...